src/components/vega-lite/index.tsx

```diff
@@ -80,8 +94,14 @@ export class VegaLite extends React.PureComponent<VegaLiteProps, {}> {
       clearTimeout(this.mountTimeout);
     }
     this.mountTimeout = setTimeout(() => {
+      this.setState({
```

Comment:
Shouldn't we set isLoading to `true` outside `setTimeout`?

(It's suppose to be pretty fast, and just in case other things block us this timeout to execute we still have isLoading ready before hand.